### PR TITLE
Build: Release PR v3.6.0

### PR DESCRIPTION
**Pull request by imnavanath**
### Description
- Final release PR. Please check all the checklist below before merging.
---

### Checklist:
- [x] Finalize changelog.txt
- [x] Add doc links in the changelog.txt
- [x] Change minimum addon update version - ASTRA_EXT_MIN_VER
- [x] Run to update packages        : `npm audit fix`
- [x] Run to produce minified files : `grunt minify`
- [x] Run to update Google Fonts    : `grunt google-fonts`
- [x] PHPCS - vendor/bin/phpcs
- [x] Run to update version number  : `grunt version-bump --ver=<version-number>`
- [x] Check the Background Process function version
- [x] Update - Tested Upto - style.css & readme.txt
- [x] Update Readme.txt file and then run: `grunt readme`
- [x] Generate POT file             : `wp i18n make-pot wp-content/themes/astra wp-content/themes/astra/languages/astra.pot`
- [ ] Add draft Release notes on github
- [ ] Add draft changelogs to wpastra.com
- [ ] Open PR to backport branch to `next-release`
---

### CODE REVIEW

**Review by sagarb90 — DISMISSED** [review]

**Review by premanshup — APPROVED** [review]

File: changelog.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-v3.6.0 (Unreleased)
+v3.6.0
 - Improvement: Added close button for multiselect selected items in Google font variants.
```
Comment (sagarb90) [inline changelog.txt:2]: @imnavanath Are we miss thsee changelog to add?

```
New: Performance – Load Google Font locally. 
New: Performance – Preload locally hosted Google Font.
Improvement: added SVG logo height option to resolve the CLS issue. 
Improvement: Improved new padding default values & GB outline buttons compatibility added.

```

Comment (imnavanath) [reply changelog.txt:2]: Just got this, on my end - updated as well locally.
```
New: Performance – Load Google Font locally. 
New: Performance – Preload locally hosted Google Font.
Improvement: added SVG logo height option to resolve the CLS issue. 
```

File: changelog.txt [hunk 2]
```diff
@@ -1,11 +1,14 @@
-v3.6.0 (Unreleased)
+v3.6.0
+- New: Performance - Load Google Font locally. (https://wpastra.com/docs/self-hosted-google-fonts/)
+- New: Performance - Preload locally hosted Google Font. (https://wpastra.com/docs/self-hosted-google-fonts/)
+- Improvement: added SVG logo height option to resolve the CLS issue.
```
Comment (premanshup) [inline changelog.txt:4]:
```suggestion
- Improvement: Added SVG logo height option to resolve the CLS issue.
```

Comment (imnavanath) [reply changelog.txt]: Done 👍

**Pull request merged**